.evergreen/evergreen.yml.in

```
@@ -1548,7 +2123,7 @@ buildvariants:
     run_on: <% out(variant.runOn) %>
     tags: <% out(variant.tags) %>
     expansions:
-      executable_os_id: <% out(variant.executableOsId) %>
+      executable_os_id: <% out(variant.executableOsId) || '' %>
```

You're probably going for

```suggestion
      executable_os_id: <% out(variant.executableOsId || '') %>
```

here, but in which cases is this actually undefined?